rust_dev_preview/iam/README.md

```diff
@@ -1,58 +1,72 @@
-# AWS SDK for Rust code examples for IAM
+# IAM code examples for AWS SDK for Rust
```

Comment:
AWS SDK for Rust --> the SDK for Rust 

(OK to use short SDK names in titles)

Reply:
fixed